environment.yml

```diff
@@ -12,6 +12,8 @@ dependencies:
   - jupyter_contrib_nbextensions
   - importlib-metadata
   - rdkit
+  # For sphinx
+  - mdanalysis-sphinx-theme >=1.0.1
```

Comment:
✅ just double-checked that this pulls in sphinx and all deps

https://github.com/conda-forge/mdanalysis-sphinx-theme-feedstock/blob/main/recipe/meta.yaml